Table "FRUITS" not found;

When trying to run the tests, I found this error:

``` text
Exception in thread "main" org.h2.jdbc.JdbcBatchUpdateException: Table "FRUITS" not found; SQL statement:
DROP TABLE fruits [42102-187], compiling:(/private/var/folders/k9/pgnpg2h92bv157mw70mhznk80000gn/T/form-init6355575859006062318.clj:1:124)
    at clojure.lang.Compiler.load(Compiler.java:7142)
    at clojure.lang.Compiler.loadFile(Compiler.java:7086)
    at clojure.main$load_script.invoke(main.clj:274)
    at clojure.main$init_opt.invoke(main.clj:279)
    at clojure.main$initialize.invoke(main.clj:307)
    at clojure.main$null_opt.invoke(main.clj:342)
    at clojure.main$main.doInvoke(main.clj:420)
    at clojure.lang.RestFn.invoke(RestFn.java:421)
    at clojure.lang.Var.invoke(Var.java:383)
    at clojure.lang.AFn.applyToHelper(AFn.java:156)
    at clojure.lang.Var.applyTo(Var.java:700)
    at clojure.main.main(main.java:37)
Caused by: org.h2.jdbc.JdbcBatchUpdateException: Table "FRUITS" not found; SQL statement:
DROP TABLE fruits [42102-187]
    at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:676)
    at clojure.java.jdbc$execute_batch.invoke(jdbc.clj:404)
    at clojure.java.jdbc$db_do_commands$fn__659.invoke(jdbc.clj:684)
    at clojure.java.jdbc$db_transaction_STAR_.doInvoke(jdbc.clj:587)
    at clojure.lang.RestFn.invoke(RestFn.java:425)
    at clojure.java.jdbc$db_do_commands.doInvoke(jdbc.clj:683)
    at clojure.lang.RestFn.applyTo(RestFn.java:142)
    at clojure.core$apply.invoke(core.clj:628)
    at clojure.java.jdbc$db_do_commands.doInvoke(jdbc.clj:690)
    at clojure.lang.RestFn.applyTo(RestFn.java:142)
    at clojure.core$apply.invoke(core.clj:630)
    at clojure.java.jdbc$db_do_commands.doInvoke(jdbc.clj:677)
    at clojure.lang.RestFn.invoke(RestFn.java:442)
    at conman.core_test$create_test_table.invoke(core_test.clj:19)
    at conman.core_test$eval2131$fn__2132.invoke(core_test.clj:31)
    at clojure.test$compose_fixtures$fn__7181$fn__7182.invoke(test.clj:681)
    at clojure.test$default_fixture.invoke(test.clj:674)
    at clojure.test$compose_fixtures$fn__7181.invoke(test.clj:681)
    at clojure.test$test_vars.invoke(test.clj:718)
    at clojure.test$test_all_vars.invoke(test.clj:728)
    at clojure.test$test_ns.invoke(test.clj:747)
    at clojure.core$map$fn__4245.invoke(core.clj:2559)
    at clojure.lang.LazySeq.sval(LazySeq.java:40)
    at clojure.lang.LazySeq.seq(LazySeq.java:49)
    at clojure.lang.Cons.next(Cons.java:39)
    at clojure.lang.RT.boundedLength(RT.java:1654)
    at clojure.lang.RestFn.applyTo(RestFn.java:130)
    at clojure.core$apply.invoke(core.clj:626)
    at clojure.test$run_tests.doInvoke(test.clj:762)
    at clojure.lang.RestFn.applyTo(RestFn.java:137)
    at clojure.core$apply.invoke(core.clj:624)
    at user$eval85$fn__140$fn__171.invoke(form-init6355575859006062318.clj:1)
    at user$eval85$fn__140$fn__141.invoke(form-init6355575859006062318.clj:1)
    at user$eval85$fn__140.invoke(form-init6355575859006062318.clj:1)
    at user$eval85.invoke(form-init6355575859006062318.clj:1)
    at clojure.lang.Compiler.eval(Compiler.java:6703)
    at clojure.lang.Compiler.eval(Compiler.java:6693)
    at clojure.lang.Compiler.load(Compiler.java:7130)
    ... 11 more
Tests failed.
```

Simply, the first time the tests are running, there's no fruits table.

--- a/test/conman/core_test.clj
+++ b/test/conman/core_test.clj
@@ -15,7 +15,7 @@
 (defn create-test-table []
   (sql/db-do-commands
    @conn
-   (sql/drop-table-ddl :fruits)
+   "DROP TABLE fruits IF EXISTS;"
    (sql/create-table-ddl
     :fruits
     [:id :int "DEFAULT 0"]

